### DISCUSSION TITLE
Pressable components inconsistency

### DISCUSSION THREAD

**Discussion by OmarBasem** [status-im/status-legacy#16667] [General]
Currently, we are not being consistent in our usage of the pressable components. We are using are a mix of `TouchableOpacity` and `TouchableWithoutFeedback`. Also, there is a newer pressable component called `Pressable` - from RN docs "If you're looking for a more extensive and future-proof way to handle touch-based input, check out the [Pressable](https://reactnative.dev/docs/pressable) API."

So shouldn't we migrate the pressable components to that new component?

cc @flexsurfer

**Comment by flexsurfer** [comment]
i think it depends on the component, TouchableOpacity and TouchableWithoutFeedback are fine, i don't see why we should use Pressable

so if the design says that on press we should change opacity, use TouchableOpacity then, if the design says touchable has no visual feedback use TouchableWithoutFeedback, so i don't see the problem here really , these two fit our needs
